nDeviceType: 4 #设备类型USBCAN-2A or USBCAN-2C or CANalyst-II
nDeviceInd: 0 #设备索引
nCanId: 0 # 0-->1通道 1-->2通道
walking_channel:
  chn1: 0x00000601  # front walking wheels
  chn2: 0x00000602  # rear walking wheels

steering_channel:
  chn1: 0x00000603 # front steering wheels
  chn2: 0x00000604  # rear steering wheels
# mode of motors (steering && walking)
# 0 -> position; 1 -> velocity; 2 -> current
Frames_Length: 1 # 每次发送几帧
steering_mode: 0 
walking_mode: 1
Homing_abs_encoder_data:
  fl: 25902  # corresponding impulse of steering motor (front left)
  fr: 26803  # corresponding impulse of steering motor (front right)
  rl: 28810 # corresponding impulse of steering motor (rear left)
  rr: 26141  # corresponding impulse of steering motor (rear right)
Homing_error_limit: 5
Pid_parameter: 
  P: 0.5
  I: 0.3
  D: 0.001